addons/backgrounds/preset.js

```diff
@@ -1 +1,16 @@
-module.exports = require('./dist/preset');
+function config(entry = []) {
+  return [
+    ...entry,
+    require.resolve('./dist/esm/preset/addDecorator'),
+    require.resolve('./dist/esm/preset/addParameter'),
+  ];
+}
+
+function managerEntries(entry = [], options) {
+  return [...entry, require.resolve('./dist/esm/register')];
+}
+
+module.exports = {
+  managerEntries,
+  config,
+};
```

Comment:
As it's done for other addons, `addons/backgrounds/src/preset/index.ts` can be deleted I guess?

Reply:
✅